### PR TITLE
Remove vestigial Windows support and defunct Irix support

### PR DESCRIPTION
**Pull request by jonjensen**
The documentation has said for 15+ years:

"Microsoft Windows is not (and will not be) supported."

https://www.interchangecommerce.org/docs/install.html

### CODE REVIEW

**Review by msjohns1 — COMMENTED** [review]
Generally fine. Review 2 in-line comments.

File: extensions/Interchange.pm [hunk 1]
```diff
@@ -445,11 +444,9 @@ sub get_filename {
 # Can't use that because it INSISTS on object
 # calls without returning a blessed object
 
-my $abspat = $^O =~ /win32/i ? '^([a-z]:)?[\\\\/]' : '^/';
-
 sub file_name_is_absolute {
     my($file) = @_;
-    $file =~ m{$abspat}oi ;
+    $file =~ m{^/}oi;
```
[the remark this answers is not in the document]
Comment (jonjensen) [reply extensions/Interchange.pm]: Good catch! Doesn't need the `i` flag anymore either.

File: lib/Vend/File.pm [hunk 2]
```diff
@@ -517,7 +511,7 @@ sub get_filename {
 # Can't use that because it INSISTS on object
 # calls without returning a blessed object
 
-my $abspat = $^O =~ /win32/i ? qr{^([a-zA-Z]:)?[\\/]} : qr{^/};
+my $abspat = qr{^/};
 my $relpat = qr{\.\.[\\/]};
```
[the remark this answers is not in the document]
Comment (jonjensen) [reply lib/Vend/File.pm]: Yep, another good catch. Removed.

**Comment by jonjensen** [comment]
@msjohns1 I noticed some other obvious old Windows junk right there and removed it. But that was more invasive so I added some testing to the unit test catalog. Please see the new commits.

**Comment by jonjensen** [comment]
@msjohns1 In addition to the earlier commits added here, today I found and removed more cruft, so please see more new commits here. Thanks.

**Review by msjohns1 — APPROVED** [review]
No issues.

**Comment by jonjensen** [comment]
@msjohns1 Thanks for the review.

**Pull request closed without merging**